forms-flow-api/src/formsflow_api/schemas/form_history_logs.py

```diff
@@ -19,9 +19,14 @@ class Meta:  # pylint: disable=too-few-public-methods
     major_version = fields.Int(data_key="majorVersion")
     minor_version = fields.Int(data_key="minorVersion")
     version = fields.Method("get_combined_version", dump_only=True)
+    isMajor = fields.Method("get_is_major", dump_only=True)
 
     def get_combined_version(self, obj):
         """Combine major and minor versions."""
         major_version = obj.major_version or 1
         minor_version = obj.minor_version or 0
         return f"{major_version}.{minor_version}"
+
+    def get_is_major(self, obj):
+        """Determine if the version is major."""
```

Comment:
why == 0 ?

Reply:
expecting a major version starts has a minor version 0 if all other case minor version is different based on that this key value setting

Reply:
so it reusable function or only for single purpose